src/map/battle.cpp

```diff
@@ -2920,7 +2920,7 @@ static int battle_get_weapon_element(struct Damage* wd, struct block_list *src,
 			element = sstatus->rhw.ele;
 		else
 			element = sstatus->lhw.ele;
-		if(is_skill_using_arrow(src, skill_id) && sd && sd->bonus.arrow_ele > ELE_NONE && sd->bonus.arrow_ele < ELE_ALL && weapon_position == EQI_HAND_R)
+		if(is_skill_using_arrow(src, skill_id) && sd && weapon_position == EQI_HAND_R)
```

Comment:
This causes a problem for normal attacks forcing all attacks to neutral or a specific element.

Scenario Replication:
- Summon whisper mob
- Use gunslinger equip bullet and silver bullet.
Result: No damage/miss

WM_Reverb Skill Scenario:
- No damage/miss